specification/_doc_ids/table.csv

```diff
@@ -0,0 +1,381 @@
+async-search,/guide/en/elasticsearch/reference/current/async-search.html
```

Comment:
> async-search,/guide/en/elasticsearch/reference/current/async-search.html

I think this is fine for the purposes of this PR, but I assume that for this to actually work in the client docs (and eventual API docs), we'll have to have the full URL in this file (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html for content in the old system or https://docs.elastic.co/en/integrations/apache for content in the new system)?

Reply:
Either is fine for me, let me know what works best for you!

Reply:
I can add the full URLs in a subsequent PR, no problem.